src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -79,11 +80,12 @@ function verify(): RequestMeta {
  * @return {RequestMeta}                Request object that can be passed to fetch() API
  */
 const userCreate = (keys: UserKeys): RequestMeta => {
-    const {publicKey, encryptedPrivateKey} = keys;
+    const {publicKey, encryptedPrivateKey, needsRotation} = keys;
 
     const body = {
         userPublicKey: publicKeyToBase64(publicKey),
         userPrivateKey: fromByteArray(encryptedPrivateKey),
+        userNeedsRotation: needsRotation,
```

Comment:
According to what `ironcore-id` says, the key should be `needsRotation` not `userNeedsRotation`.